feature/mpls/sr/otg_tests/isis_node_sid_forward/README.md

```diff
@@ -0,0 +1,67 @@
+# SR-1.1: Transit forwarding to Node-SID via ISIS
+
+## Summary
+
+MPLS-SR transit forwarding to Node-SID distributed over ISIS
+
+## Testbed type
+
+*  [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+### Configuration
+
+Topology: ATE1—DUT1–ATE2
+                               
+*   Configure Segment Routing Global Block (start: 400000 range: 65001)
+*   Enable Segment Routing for the ISIS
+*   Enable MPLS forwarding (if required)
```

Comment:
There shouldn't be any "if required" in this case I think?  The OC should work one way.   If a vendor does not require this, they should accept the enable leaf = true.       (although they would send an error if enable = false, assuming that they don't support explicitly disabling mpls forwarding)

Reply:
Removed.